### DISCUSSION TITLE
对表格数据行进行修改其中1行数据点击保存按钮后，仍无法再修改另外1行，提示未保存

### DISCUSSION THREAD

**Discussion by baoshizhi01** [cloudtogo/pageplug#32] [Q&A]
不需要将表格內的数据，保存更新到数据源，
只想更改当前表格的数据，让后想提取表格的当前最新数据

目前能更改其中一行数据，但点击保存按钮后保存按钮仍高亮显示，
将光标移到其他行修改时，，提示先保存，不让修改
![image](https://github.com/cloudtogo/pageplug/assets/91464547/b713b332-a283-46bf-9c1f-e5aaa5bf61e3)

**Comment by zsf1482451437** [comment]
![image](https://github.com/cloudtogo/pageplug/assets/62194517/08556c95-8ecc-4913-81d7-b562ddbab8a6)
更新有两种模式，一种是以行为单位，一种是以整个表格，我想第二种就能满足你的需求了
